### ISSUE TITLE
Keyboard navigation doesn't work if Start View is set to '2' (decade)

### ISSUE THREAD

**Issue by plcrewe** [eternicode/bootstrap-datepicker#1344]
Easily reproduced using the sandbox:-

http://eternicode.github.io/bootstrap-datepicker/?markup=input&format=&weekStart=&startDate=&endDate=&startView=2&minViewMode=0&todayBtn=false&clearBtn=false&language=en&orientation=auto&multidate=&multidateSeparator=&keyboardNavigation=on&forceParse=on#sandbox

With the decade initially displayed on the calendar control, when pressing [UpArrow], for example, the JS appears to reduce the calculated date by 1 WEEK - rather than 4 years.  It appears to always assume that DAYS are currently being displayed.

**Comment by acrobat** [comment]
This PR tries to fix the keyboard navigation on the month view #1077. If it works we can extend it so it also fixes the year view keyboard navigation

**Comment by plcrewe** [comment]
The month view fix looks like it's working - except that for me when I click [UpArrow] I don't see a highlighted cell.  That may be something going on on my machine, but it looks like I'll have to fiddle with the CSS to see which cell is being highlighted.  If you can point me in the right direction, that would really help.  Thanks.

**Comment by tommysanterre** [comment]
I am currently working on a pull request to fix this
